mhs/inbound/nhs-inbound-env-example.yaml

```diff
@@ -1,4 +1,5 @@
 #Common env variables for inbound
+MHS_SERVER_PORT: "8081"
```

Comment:
I'd made server port env variable different for each component (like `MHS_INBOUND_SERVER_PORT`) so they won't clash with each other.